src/main/java/Duke.java

```diff
@@ -1,10 +1,205 @@
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        printResponse(String.format(OperationsMessages.INITIAL_MSG, BOT_NAME));
+        Scanner scanner = new Scanner(System.in);
+        while (scanner.hasNextLine()) {
+            String line = scanner.nextLine();
+            try {
+                handleInput(line);
+            } catch (EmptyInputException e) {
+                // TODO Auto-generated catch block
+                // e.printStackTrace();
+                printResponse(ErrorMessages.EMPTY_INPUT);
+            } catch (InvalidCommand e) {
+                printResponse(ErrorMessages.INVALID_COMMAND);
+            } catch (ArrayIndexOutOfBoundsException e) {
+                printResponse(ErrorMessages.INCORRECT_FIELDS);
+            } catch (IOException e) {
+                printResponse(ErrorMessages.LOAD_FAIL);
+            }
+        }
+        scanner.close();
+    }
+
+    private static void handleInput(String line) throws EmptyInputException, InvalidCommand, IOException {
+        String[] lineParts = line.split(" ");
+        String command = lineParts[0];
+        if (command.equals("list")) {
+            printItems();
+        } else if (command.equals("save")) {
+            save(saveFilePath);
+            printResponse(OperationsMessages.SAVED_MSG);
+        } else if (command.equals("load")) {
+            load(saveFilePath);
+            printResponse(OperationsMessages.LOADED_MSG);
+        } else if (lineParts.length <= 1) {
+            // dont move on if less than 2 parts
+            throw new EmptyInputException();
+        } else if (command.equals("bye")) {
+            printResponse(OperationsMessages.BYE_MSG);
+        } else if (command.equals("todo")) {
+            line = line.substring(command.length() + 1);
+            addItem(new Todo(line));
+        } else if (command.equals("delete") || command.equals("remove")) {
+            line = line.substring(command.length() + 1);
+            removeItem(Integer.parseInt(lineParts[1]));
+        } else if (command.equals("deadline")) {
+            line = line.substring(command.length() + 1);
+            createDeadline(line);
+        } else if (command.equals("event")) {
+            line = line.substring(command.length() + 1);
+            createEvent(line);
+        } else if (command.equals("list")) {
+            printItems();
+        } else if (command.equals("mark")) {
+            // String item = line.substring(4); // if want to mark by name
+            markItem(Integer.parseInt(lineParts[1]));
+        } else if (command.equals("unmark")) {
+            // String item = line.substring(6); // if want to unmark by name
+            unmarkItem(Integer.parseInt(lineParts[1]));
+        } else { // add item
```

Comment:
Java coding standard: Comments should be indented relative to their position in the code.

Can consider moving it to the next line.